agents/tools/read_file_structure.py

```
@@ -36,7 +36,7 @@ def cached_dirs(self) -> list[Path]:
         # Ensure they are sorted by depth for is_subsequence logic
         return sorted(dirs, key=lambda x: len(x.parts))
 
-    def _run(self, dir: str) -> str:
+    def _run(self, dir: str = ".") -> str:
```

Ruff (0.15.17)

[error] Function argument `dir` is shadowing a Python builtin

(A002)